### PR TITLE
update r8 tools

### PR DESCRIPTION
**Pull request by fadeltd**
Fixing this issue https://github.com/google/bundletool/issues/113

### CODE REVIEW

**Comment by fadeltd** [comment]
@googlebot I signed it!

File: build.gradle [hunk 1]
```diff
@@ -32,25 +32,28 @@ configurations {
 
 // The repackaging rules are defined in the "shadowJar" task below.
 dependencies {
-    compile "com.android.tools:r8:1.0.37"
+    compile "com.android.tools:r8:1.4.93"
```
Comment (JakeWharton) [inline build.gradle]: 1.5 will be released this week with AS 3.5 stable so you might want to wait a day or two here.

Comment (fadeltd) [reply build.gradle]: AS 3.5 is out now, I checked r8 still no new update. ok, I'll wait and update later. For the time being, I'll just use my own executableJar

Comment (ganadist) [reply build.gradle]: You can use R8 1.5.64 (bundled on AS 3.5.0) with following dependency.

```
diff --git a/build.gradle b/build.gradle
index 58411da..79e3684 100755
--- a/build.gradle
+++ b/build.gradle
@@ -22,6 +22,9 @@ repositories {
     mavenLocal()
     jcenter()
     google()
+    maven {
+        url "http://storage.googleapis.com/r8-releases/raw"
+    }
 }
 
 configurations {
@@ -32,7 +35,7 @@ configurations {
 
 // The repackaging rules are defined in the "shadowJar" task below.
 dependencies {
-    compile "com.android.tools:r8:1.0.37"
+    compile "com.android.tools:r8:1.5.64"
     compile "com.android.tools.build:apkzlib:3.4.0-beta01"
     compile "com.android.tools.ddms:ddmlib:26.2.0"
 
```

Comment (fadeltd) [reply build.gradle]: > You can use R8 1.5.64 (bundled on AS 3.5.0) with following dependency.
> 
> ```
> diff --git a/build.gradle b/build.gradle
> index 58411da..79e3684 100755
> --- a/build.gradle
> +++ b/build.gradle
> @@ -22,6 +22,9 @@ repositories {
>      mavenLocal()
>      jcenter()
>      google()
> +    maven {
> +        url "http://storage.googleapis.com/r8-releases/raw"
> +    }
>  }
>  
>  configurations {
> @@ -32,7 +35,7 @@ configurations {
>  
>  // The repackaging rules are defined in the "shadowJar" task below.
>  dependencies {
> -    compile "com.android.tools:r8:1.0.37"
> +    compile "com.android.tools:r8:1.5.64"
>      compile "com.android.tools.build:apkzlib:3.4.0-beta01"
>      compile "com.android.tools.ddms:ddmlib:26.2.0"
>  
> ```

Could not find com.android.tools:r8:1.5.64

Comment (JakeWharton) [reply build.gradle]: It's 1.5.68, and it's out now

File: build.gradle [hunk 2]
```diff
@@ -36,22 +37,29 @@ dependencies {
...
     compileLinux "com.android.tools.build:aapt2:3.5.0-alpha03-5252756:linux"
 
+    testCompileOnly "com.google.auto.value:auto-value-annotations:1.6.5"
+    testAnnotationProcessor "com.google.auto.value:auto-value:1.6.5"
+
     testCompile "com.android.tools.build:aapt2-proto:0.4.0"
     testCompile "com.google.auto.value:auto-value-annotations:1.6.2"
     testAnnotationProcessor "com.google.auto.value:auto-value:1.6.2"
     testCompile "com.google.errorprone:error_prone_annotations:2.3.1"
+    testCompile "com.google.errorprone:error_prone_annotations:2.3.1"
```
Comment (rallat) [inline build.gradle]: Why is this one duplicated?

Comment (fadeltd) [reply build.gradle]: It was my mistake when resolving conflicts

**Comment by rallat** [comment]
For us 0.10.3 fixed the #113 https://github.com/google/bundletool/issues/113#issuecomment-533451423

**Pull request closed without merging**